src/test/java/ui/main_menu/MainMenuTest.java

```diff
@@ -0,0 +1,120 @@
+package ui.main_menu;
+
+import game.ui.SceneLauncher;
+import game.ui.game.GameScene;
+import game.ui.menu.MenuScene;
+import org.assertj.swing.fixture.FrameFixture;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import ui.util.TestComponentNames;
+
+import static org.junit.Assert.assertEquals;
+
+public class MainMenuTest {
+    private FrameFixture window;
+    private MenuScene menuScene;
+    private SceneLauncher launcher;
+
+    @Before
+    public void setUp() {
```

Comment:
Personally, I'd add the buttons as class variables for better code quality, but other than that looks good.

Reply:
Refactored, added as private variables